pinot-core/src/main/java/org/apache/pinot/core/auth/Actions.java

```diff
@@ -0,0 +1,144 @@
+    public static final String DELETE_SEGMENT = "DeleteSegment";
+    public static final String DELETE_TABLE = "DeleteTable";
+    public static final String DELETE_TIME_BOUNDARY = "DeleteTimeBoundary";
+    public static final String DISABLE_TABLE = "DisableTable";
+    public static final String DOWNLOAD_SEGMENT = "DownloadSegment";
+    public static final String ENABLE_TABLE = "EnableTable";
+    public static final String FORCE_COMMIT = "ForceCommit";
+    public static final String GET_BROKER = "GetBroker";
+    public static final String GET_CONFIG = "GetConfig";
+    public static final String GET_CONSUMING_SEGMENTS = "GetConsumingSegments";
+    public static final String GET_CONTROLLER_JOBS = "GetControllerJobs";
+    public static final String GET_DEBUG_INFO = "GetDebugInfo";
+    public static final String GET_EXTERNAL_VIEW = "GetExternalView";
+    public static final String GET_IDEAL_STATE = "GetIdealState";
+    public static final String GET_INSTANCE = "GetInstance";
+    public static final String GET_INSTANCE_PARTITIONS = "GetInstancePartitions";
+    public static final String GET_METADATA = "GetMetadata";
+    public static final String GET_PAUSE_STATUS = "GetPauseStatus";
+    public static final String GET_ROUTING_TABLE = "GetRoutingTable";
+    public static final String GET_SCHEMA = "GetSchema";
+    public static final String GET_SEGMENT = "GetSegment";
+    public static final String GET_SEGMENT_LINEAGE = "GetSegmentLineage";
+    public static final String GET_SEGMENT_MAP = "GetSegmentMap";
+    public static final String GET_SERVER_MAP = "GetServerMap";
+    public static final String GET_SIZE = "GetSize";
+    public static final String GET_STATE = "GetState";
+    public static final String GET_STORAGE_TIER = "GetStorageTier";
+    public static final String GET_TABLE_CONFIG = "GetTableConfig";
+    public static final String GET_TABLE_LEADER = "GetTableLeader";
+    public static final String GET_TIME_BOUNDARY = "GetTimeBoundary";
+    public static final String PAUSE_CONSUMPTION = "PauseConsumption";
+    public static final String QUERY = "Query";
+    public static final String REBALANCE_TABLE = "RebalanceTable";
+    public static final String REBUILD_BROKER_RESOURCE = "RebuildBrokerResource";
+    public static final String REFRESH_ROUTING = "RefreshRouting";
+    public static final String RELOAD_SEGMENT = "ReloadSegment";
+    public static final String REPLACE_SEGMENT = "ReplaceSegment";
+    public static final String RESUME_CONSUMPTION = "ResumeConsumption";
+    public static final String UPDATE_INSTANCE_PARTITIONS = "UpdateInstancePartitions";
+    public static final String UPDATE_SCHEMA = "UpdateSchema";
+    public static final String UPDATE_TABLE_CONFIG = "UpdateTableConfig";
+    public static final String UPDATE_TABLE_CONFIGS = "UpdateTableConfigs";
+    public static final String UPLOAD_SEGMENT = "UploadSegment";
+    public static final String VALIDATE_CONFIG = "ValidateConfig";
```

Comment:
For consistency, suggest rename this to `VALIDATE_TABLE_CONFIGS`

Reply:
We have GET_TABLE_CONFIG, GET_TABLE_CONFIGS and UPDATE_TABLE_CONFIG and UPDATE_TABLE_CONFIGS, shall we have one version only, i.e.
GET_TABLE_CONFIG
UPDATE_TABLE_CONFIG

and

remove GET_TABLE_CONFIGS and UPDATE_TABLE_CONFIGS ?